src/Ennemies/Vampire.java

```diff
@@ -0,0 +1,21 @@
+package Ennemies;
+
+public class Vampire extends Enemy  {
+
+    public Vampire(){
+        super();
+    }
+
+
+    @Override
+    public int attack() {
+        int inflicted = RANDOM.nextInt(MAXIMUM_ATTACK_DAMAGE);
```

Comment:
Would you not want to simply return the `RANDOM` value? Why are we storing this in a `inflicted` variable?

Reply:
It could be used to give the Vampire a special ability like inflicted/3 of health regenerated